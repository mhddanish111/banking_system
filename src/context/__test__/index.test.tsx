import { shallow } from 'enzyme';
import AppContextWrapper from '../index';

describe('Context', () => {
  const wrapper = shallow(<AppContextWrapper />);
  test('AppContextWrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
