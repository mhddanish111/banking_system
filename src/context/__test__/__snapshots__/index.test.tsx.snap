// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Context AppContextWrapper 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "state": Object {
        "accounts": Array [],
        "cards": Array [],
        "user": Object {},
        "userProfile": Object {},
      },
    }
  }
/>
`;
